neptune/internal/utils/alpha_integration.py

```diff
@@ -13,38 +13,82 @@
 # See the License for the specific language governing permissions and
 # limitations under the License.
 #
+import abc
+from collections import namedtuple
 
 from neptune.alpha import types as alpha_types
+from neptune.alpha.attributes import constants as alpha_consts
 from neptune.alpha.internal import operation as alpha_operation
 from neptune.alpha.internal.backends.api_model import AttributeType as AlphaAttributeType
 from neptune.exceptions import NeptuneException
 from neptune.internal.channels.channels import ChannelType, ChannelValueType
 
+# Alpha equivalent of old api's `KeyValueProperty` used in `Experiment.properties`
+AlphaKeyValueProperty = namedtuple('AlphaKeyValueProperty', ['key', 'value'])
 
-class AlphaChannelDTO:
-    """It's simple wrapper for `AttributeDTO` objects which uses alpha series attributes to fake channels.
 
-    Alpha leaderboard doesn't have `ChannelDTO` since it doesn't support channels at all,
-    so we do need fake `ChannelDTO` class for backward compatibility with old client's code."""
+class AlphaAttributeWrapper(abc.ABC):
+    """It's simple wrapper for `AttributeDTO`."""
 
-    _allowed_atribute_types = [
-        AlphaAttributeType.FLOAT_SERIES.value,
-        AlphaAttributeType.STRING_SERIES.value,
-        AlphaAttributeType.IMAGE_SERIES.value,
-    ]
+    _allowed_atribute_types = list()
 
     def __init__(self, attribute):
         """Expects `AttributeDTO`"""
-        if not self.is_valid_attribute_for_channel(attribute):
+        assert self._allowed_atribute_types is not None
+        if not self.is_valid_attribute(attribute):
             raise NeptuneException(f"Invalid channel attribute type: {attribute.type}")
 
         self._attribute = attribute
 
     @classmethod
-    def is_valid_attribute_for_channel(cls, attribute):
+    def is_valid_attribute(cls, attribute):
         """Checks if attribute can be used as channel"""
         return attribute.type in cls._allowed_atribute_types
 
+    @property
+    def _properties(self):
+        """Returns proper attribute property according to type"""
+        return getattr(self._attribute, f'{self._attribute.type}Properties')
+
+
+class AlphaPropertyDTO(AlphaAttributeWrapper):
+    """It's simple wrapper for `AttributeDTO` objects which uses alpha variables attributes to fake properties.
+
+    Alpha leaderboard doesn't have `KeyValueProperty` since it doesn't support properties at all,
+    so we do need fake `KeyValueProperty` class for backward compatibility with old client's code."""
+
+    _allowed_atribute_types = [
+        AlphaAttributeType.FLOAT.value,
+        AlphaAttributeType.STRING.value,
+        AlphaAttributeType.DATETIME.value,
```

Comment:
In contrast to parameters, properties was strings only in old Neptune.

Reply:
Right, I didn't notice this at first, and then I forgot to fix it in this place...